Tests/TestMacros/MacroTests.swift

```diff
@@ -1,35 +1,42 @@
-import Foundation
+import XCTest
 
-class MacroTests: MockoloTestCase {
-   func testMacroInFunc() {
+final class MacroTests: MockoloTestCase {
+    func testMacroInFunc() {
         verify(srcContent: macroInFunc,
                dstContent: macroInFuncMock)
     }
 
+    func testMacroInFuncWithOverload() {
+        XCTExpectFailure("Resolving overloading in #if is broken.") {
```

Comment:
In the process of refactoring, I discovered a new bug🤣

Reply:
New finding! let's create an issue.